src/target/imxrt.c

```diff
@@ -463,8 +463,8 @@ static void imxrt_spi_wait_complete(target_s *const target)
 #ifdef ENABLE_DEBUG
 		/* Read out the status code and display it */
 		const uint32_t status = target_mem_read32(target, IMXRT_FLEXSPI1_STAT1);
-		DEBUG_TARGET(
-			"Error executing sequence, offset %u, error code %u\n", (status >> 16U) & 0xfU, (status >> 24U) & 0xfU);
+		DEBUG_TARGET("Error executing sequence, offset %" PRIu32 ", error code %" PRIu32 "\n", (status >> 16U) & 0xfU,
```

Comment:
In the interests of keeping Flash usage down, perhaps it would be better to cast the shifted status values to uint8_t's?

Reply:
I've made the change. Please let me know if that affects flash at all.

Reply:
From the CI run: ```
Run head/scripts/diff_size.py base/build.log head/build.log
rom: +0 / 131072 B (+0.00%)
ram: +0 / 20480 B (+0.00%)
```
So that seems perfect